[WARNING] [2021-06-15 21:56:46,864] [3074:139995760379712] - data_io.py[line:855]: DataIO is deprecated, and will be removed in 1.7, use DataTransform module instead
[ERROR] [2021-06-15 21:56:52,087] [3074:139995760379712] - task_executor.py[line:191]: fit & transform data' header should be same
Traceback (most recent call last):
  File "/fate/python/fate_flow/operation/task_executor.py", line 168, in run_task
    run_object.run(component_parameters_on_party, task_run_args)
  File "/fate/python/federatedml/model_base.py", line 98, in run
    this_data_output = func(*real_param)
  File "/fate/python/federatedml/util/data_io.py", line 895, in transform
    return self.reader.read_data(data_inst, "transform")
  File "/fate/python/federatedml/util/data_io.py", line 150, in read_data
    data_instance = data_overview.header_alignment(data_instance, fit_header)
  File "/fate/python/federatedml/statistic/data_overview.py", line 75, in header_alignment
    raise ValueError("fit & transform data' header should be same")
ValueError: fit & transform data' header should be same
[WARNING] [2021-06-15 21:57:04,281] [3233:140059497649984] - data_io.py[line:855]: DataIO is deprecated, and will be removed in 1.7, use DataTransform module instead
[WARNING] [2021-06-15 21:57:11,797] [3323:140186488915776] - intersect_param.py[line:63]: 'EncodeParam' will be renamed to 'RawParam' in future release.Please do not rely on current param naming in application.
